druid-shell/src/platform/windows/mod.rs

```diff
@@ -135,11 +135,11 @@ pub struct IdleHandle {
 }
 
 trait IdleCallback: Send {
-    fn call(self: Box<Self>, a: &mut dyn Any);
+    fn call(self: Box<Self>, a: &dyn Any);
```

Comment:
Hmm, I think we might want to standardize on `&mut` for this. That's basically saying that it will be called single-threaded. On the other hand, changing to that later is easier than the other way around.

Reply:
Yea, that was my thinking; making this non-mut didn't break anything, and we can change it later. In any case this gets moved again in #322, so I'd like to punt on this for now?